config/features.yaml

```diff
@@ -61,3 +61,19 @@ github_accounts:
         aggregate: True
         tags: ['Github Account']
         emojis: ['ID_BUTTON']
+
+ip_address:
+        query_string: 'source_short:"LOG"'
+        attribute: 'ipv4'
```

Comment:
ok... so "source_short:"LOG"' could result in quite a lot of events selected.... but not nearly all of those have an ipv4 attribute... is it possible to change the query_string to something like:

"""
query_string: 'source_short:"LOG" AND _exists_:"ipv4"'
"""

That way you should only get back the events that at least have that attribute set.